examples/light-switch-app/light-switch-common/light-switch-app.zap

```diff
@@ -4402,31 +4402,31 @@
     },
     {
       "id": 2,
-      "name": "MA-onofflightswitch",
+      "name": "MA-dimmerswitch",
```

Comment:
Just raising attention that this is a common zap file for a few other platform light-switch app. They might be ok with this addition.
If not we can create zap file specific for silab's light-switch app.